src/client/vuecreatejob.js

```diff
@@ -0,0 +1,206 @@
+
+// loads Jobs
+// const server = "https://masteringjs-job-board.azurewebsites.net";
+const server = "http://localhost:7071";
+const payment = server+'/api/stripeCheckout';
+
+const app = new Vue({
+  data: () => ({
+    company: null,
+    logo: null,
+    title: null,
+    location: 'Anywhere',
+    sticky: false,
+    description: null,
+    url: null,
+    instructions: null,
+    email: null,
+    feedback: null,
+    invoiceAddress: null,
+    invoiceNotes: null,
+    tags: 'like,this',
+    displayImage: true,
+    previewImage: null,
+    loading: false,
+    isActive: false
+  }),
+  updated() {
+    console.log(new Date(), 'State Change:', this.$data);
+  },
+  mounted() {
+    window.$saveState = () => {
+      window.localStorage.setItem('__state', JSON.stringify(this.$data));
+    };
+    window.$loadState = () => {
+      const saved = JSON.parse(window.localStorage.getItem('__state'));
+      if (!saved) {
+        throw new Error('No saved data!');
+      }
+      Object.assign(this.$data, saved);
+    };
+    window.$clearState = () => {
+      window.localStorage.setItem('__state', '');
+    };
+  },
+  methods: {
+    async postJob() {
+      this.loading = true;
+      this.tags = this.tags.split(",");
+     let clientid = await axios.post(server + '/api/createJob', {
+        company: this.company,
+        logo: this.logo,
+        title: this.title,
+        location: this.location,
+        email: this.email,
+        tags: this.tags,
+        sticky: this.sticky,
+        description: marked(this.description),
+        url: this.url,
+        instructions: this.instructions,
+        feedback: this.feedback,
+        invoiceAddress: this.invoiceAddress,
+        invoiceNotes: this.invoiceNotes,
+      }).then((response) => {return response.data.job._id});
+      console.log('Done');
+      var stripe = Stripe('pk_test_51IkuAqIFSwo5WpGWudAKEeemrymI6EmICEAgkgvlq4Bo5jJ1uuMRlrBRw9kvHH7boANqjE7Y6Mb7lQmsXRQoZo3x00Ek1L6d8A');
```

Comment:
Move the `Stripe()` declaration out of the component and into [global scope](https://masteringjs.io/tutorials/fundamentals/global-variable)